cli/pkg/workspace/airy_yaml.go

```diff
@@ -5,10 +5,10 @@ type KubernetesConf struct {
 	ContainerRegistry string `yaml:"containerRegistry"`
 	Namespace         string `yaml:"namespace"`
 	NgrokEnabled      bool   `yaml:"ngrokEnabled"`
-	Host              string `yaml:"host"`
 }
 
 type IngressConf struct {
+	Host                    string            `yaml:"host,omitempty"`
```

Comment:
Host is moved to the `ingress` section, as the items in the `kubernetes` section will soon be removed.